### PR TITLE
Add main game implementation

### PR DESCRIPTION
**Pull request by vbeardow**
Add main game implementation in game.py, controlled by variables playing and game_on in game_config.

Add function and tests to ask player if they want to continue playing.

Update some of the functions and associated tests such that while loop does not break when invalid input is entered, but instead prints a statement until user enters a valid input.

### CODE REVIEW

File: blackjack/actions.py [hunk 1]
```diff
@@ -43,8 +46,27 @@ def reveal_cards(my_hand: Hand, reveal_all=True) -> None:
...
             print(str(card))
     else:
         print(str(my_hand.cards[0]))
+
+
+def continue_playing() -> None:
+    """Ask player if they want to continue playing (input y or n)
+
+    Raises:
+        Exception: Exception raised when user has entered something other than y or n.
```
Comment (Edward-Boulderstone) [inline blackjack/actions.py]: Need to update your docstring

File: blackjack/actions.py [hunk 2]
```diff
@@ -23,14 +24,16 @@ def stick_or_twist(hand: Hand, deck: Deck) -> None:
...
+        move = input("Stick or twist? Input s or t: ")
+
         if move == "t":
             twist(hand, deck)
         elif move == "s":
             print("Player has chosen to stick.")
+            game_config.playing = False
         else:
-            raise Exception("Please enter either s or t")
+            print("Invalid input. Please enter either s or t.")
```
Comment (Edward-Boulderstone) [inline blackjack/actions.py]: I Feel like you have some duplicated code here, in asking for user input, would it be better to move that into a seperate function?

File: blackjack/actions.py (hunk 1 above)
Comment (vbeardow) [reply blackjack/actions.py]: Updated

File: blackjack/actions.py (hunk 2 above)
Comment (vbeardow) [reply blackjack/actions.py]: Have moved a part of this into a separate function

**Pull request merged**